web/router.ex

```diff
@@ -22,7 +22,10 @@ defmodule CodeCorps.Router do
   scope "/", CodeCorps, host: "api." do
     pipe_through :api
 
+    get "/users/email_available", UserController, :email_available
+    get "/users/username_available", UserController, :username_available
     resources "/users", UserController, except: [:new, :edit]
+
```

Comment:
If it goes bellow, then the matcher matches with "GET /users/:id", where id is "email_available" or "username_available"